daemon/algod/api/algod.oas2.json

```diff
@@ -726,7 +726,8 @@
             "in": "body",
             "required": true,
             "schema": {
-              "type": "string"
+              "type": "string",
+              "format": "binary"
```

Comment:
Why binary? it is text source.
And seems like `binary` format is [not supported by oapi-codegen](https://github.com/deepmap/oapi-codegen/blob/master/pkg/codegen/schema.go#L260).

Reply:
It is handy to distinguish string (type) from in-body with extra tag (format) binary so that the getter will return a byte array and not string for writing to the client body. 

This is similar to: 
            "description": "The byte encoded signed transaction to broadcast to network",
            "name": "rawtxn",
            "in": "body",
            "required": true,
            "schema": {
              "type": "string",
              "format": "binary"
            }

Reply:
got it, thanks